index.html

```diff
@@ -22,91 +23,116 @@
-        <img src="images/detect.png" alt="Detection Process" class="img-center">
+        <h3 class="pt-5">Building your application</h3>
 
-        <h3 class="mt-5 pt-4">Analyze</h3>
-        <p>If an existing image is present, metadata about the last build is recovered from the config layer. During the build step, buildpacks may choose to keep, replace, or delete individual directories based on the metadata. These directories are stored as layers.</p>
-        <img src="images/analyze.png" alt="Analyze Process" class="img-center">
+        <p>Let’s get started building your first application with Cloud Native Buildpacks.</p>
 
-        <h3 class="mt-5 pt-4">Build</h3>
-        <p>Each buildpack examines layer metadata from the previous build, re-generates any layer directories that need to be replaced, and updates metadata (if necessary). Buildpacks may choose to modify the app source directory. A transparent, inter-build cache may be used to improve performance and provide other buildpacks with build-time dependencies.</p>
-        <img src="images/build.png" alt="Build Process" class="img-center">
+        <p>Build your application by passing in your application source code directory and image name to the <code class="highlight">pack build</code> command.</p>
 
-        <h3 class="mt-5 pt-4">Export</h3>
-        <p>The remote app layer and any re-generated buildpack layers are replaced. The config layer is updated with new metadata. This process uses image layer rebasing, which can be accomplished remotely against a Docker v2 registry.</p>
-        <img src="images/export.png" alt="Export Process" class="img-center">
+        <pre class="command-line code-block"><code>pack build --path &lt;path to source code&gt; &lt;repository name&gt;:&lt;image tag (optional)&gt;</code></pre>
 
-        <h3 class="mt-5 pt-4">Security Patching</h3>
-        <p>When CVEs hit, certain image layers can be patched without changing app behavior. These layers can be rebased en-mass using remote image layer rebasing on recent Docker registries. This is a metadata change that requires minimal data transfer.</p>
+        <p>Upon a successful build, you will see your image in your local Docker daemon.</p>
 
-        <p>Using this technique, CVEs in dependencies with strong compatibility guarantees (e.g., patches to OpenSSL, patches to the JVM) can be mitigated rapidly for many apps.</p>
-        <img src="images/cve.png" alt="Security Patching" class="img-center">
+        <h3 class="pt-5">Running Your Application</h3>
+        <p>Now that you have successfully built your OCI image using Cloud Native Buildpacks you can run your application using the Docker CLI.</p>
 
+        <pre class="command-line code-block"><code>docker run -e PORT=8000 -p 8000:8000 &lt;image name&gt;</code></pre>
```

Comment:
@jkutner Any reason we're modifying this in the example, instead of leaving it at `8080`?

(If we don't like 8080 everywhere, I think we should introduce a build-time parameter to set it in the final image.)

Reply:
No reason. Should be 8080